.github/workflows/on-code-change.yml

```diff
@@ -39,8 +39,8 @@ jobs:
       - uses: actions/checkout@v4
       - uses: jahia/jahia-modules-action/build@v2
         with:
-          tests_module_type: "javascript"
-          tests_module_path: "jahia-test-module/"
+          # the test module is built as part of the Maven build
```

Comment:
Could that cause an issue to have the test module built as regular module of the Maven multi-module project? In other words, what was the benefit of building it separately in the CI (with the extra complexity added)? 🤔

Reply:
@dgriffon / @jkevan , any concerns with that? Or are you OK to merge it as-is? Thanks!

Reply:
Ok for me as the test module now follows the version of the project.